sources/packages/backend/libs/sims-db/src/entities/disbursement-overaward-origin.type.ts

```diff
@@ -10,20 +10,14 @@ export enum DisbursementOverawardOriginType {
   /**
    * Part of an overaward balance was used to change an award value.
    */
-  AwardValueAdjusted = "Award value adjusted",
-  /**
-   * An pending award that included an overaward was cancelled and
-   * the overaward value must to be added back to the overaward balance.
-   * @deprecated to be removed.
-   */
-  PendingAwardCancelled = "Pending award cancelled",
+  AwardDeducted = "Award deducted",
```

Comment:
I think we need to update the comment as we are more specific here to 'deducted' now. Also, are we going to to have an "AwardAdded"?

Reply:
IMO the existing comment on AwardDeducted(this is award deducted to pay the overaward balance) seems to be fine. We can adjust if required in upcoming PR.